src/database_types_file.ts

```diff
@@ -22,11 +22,13 @@ export interface Database {
           extra_information: string
           family_id: string
           feminine_products: string[]
+          flag_attention: boolean
           full_name: string
           other_items: string[]
           pet_food: string[]
           phone_number: string
           primary_key: string
+          signposting_call: boolean
```

Comment:
Should the fields `flag_attention` and `signposting_call` not be for each individual parcel? Also, would field names such as `flagged_for_attention` or `signposting_call_required` be a bit more clear?

Reply:
sure will change